src/CommunityToolkit.Mvvm.Fixers/FieldReferenceForObservablePropertyFieldFixer.cs

```diff
@@ -15,57 +13,74 @@
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
 
 namespace CommunityToolkit.Mvvm.Fixers;
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-[ExportCodeFixProvider(LanguageNames.CSharp), Shared]
-public class FieldReferenceForObservablePropertyFieldFixer : CodeFixProvider
+/// <summary>
+/// A code fixer that automatically updates references to fields with <c>[ObservableProperty]</c> to reference the generated property instead.
+/// </summary>
```

Comment:
Frankly, docs on these are probably not a good idea. They're not really a public API, they don't need to be documented.

Reply:
Everything is documented, regardless of visibility. It's also for consistency and future reference. Also helps my OCD.